src/components/Proposals/TargetAppBadge.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { AppBadge, GU } from '@aragon/ui'
+import { useDescribeVote } from '../../hooks/useDescribeVote'
+import { useOrgApps } from '../../providers/OrgApps'
+import { getAppPresentation } from '../../utils/app-utils'
+import LoadingSkeleton from '../Loading/LoadingSkeleton'
+
+function TargetApp({ vote }) {
+  const {
+    emptyScript,
+    targetApp,
+    loading: descriptionLoading,
+  } = useDescribeVote(vote.script, vote.id)
```

Comment:
We should generally try to only pass the props needed by a component, I think here is a good chance to pass just `voteScript` and `voteId` as props.